packages/yum/test.sh

```diff
@@ -90,7 +90,15 @@ REPO
          https://repo.percona.com/yum/percona-release-latest.noarch.rpm
     percona_package_version=$(echo ${mysql_version} | sed -e 's/\.//g')
     old_package=${package}
-    sudo percona-release setup ps${percona_package_version}
+    if [ "${percona_package_version}" = "80" ]; then
+      sudo percona-release setup ps${percona_package_version}
+    else
+      # TODO: We can enable this after we release Mroonga 15.03.
+      # We don't release Mroonga for Percona Server 8.4 yet.
+      # So we can't install the previous Mroonga that depends on Percona Server 8.4.
+      old_package=
+      sudo percona-release enable-only ps-${percona_package_version}-lts release
```

Comment:
PGroongaではここらへんを自動化していたと思うんですが、Mroongaでも同じようにはできませんか？

Reply:
自動でスキップするようにしました。

https://github.com/komainu8/mroonga/actions/runs/13625464343/job/38081993841

```
 + is_first_release=yes
  + '[' branch = tag ']'
  + '[' yes = yes ']'
  + echo 'Skip because percona-server-8.4-mroonga hasn'\''t been released yet.'
  Skip because percona-server-8.4-mroonga hasn't been released yet.
```